### ISSUE TITLE
Results from Galaxy test tool workflows are not displayed correctly by Refinery

### ISSUE THREAD

**Issue by hackdna** [parklab/refinery-platform#476]
Workflow runs and resulting file is imported into Refinery but not displayed in the dataset view. This is likely caused by the tool output handling changes in the latest version of galaxy-dist.

Relevant Refinery logs entries:
```
2015-01-13 13:22:01 DEBUG    tasks download_history_files: analysis_manger.download_history_files called
2015-01-13 13:22:01 INFO     tasks create: Creating FileStoreItem using source 'http://192.168.50.1:8080/datasets/fb85969571388350/display?to_ext=txt'
2015-01-13 13:22:01 INFO     models set_filetype: 'galaxy.datatypes.data.text' is an unknown file type
...
2015-01-13 13:22:04 WARNING  models get_item: FileStoreItem with UUID 'None' does not exist
2015-01-13 13:22:04 WARNING  search_indexes prepare: Unable to get file store item None. No file type available.
2015-01-13 13:22:05 INFO     utils _index_annotated_nodes: 1 nodes indexed in 0.3341588974
2015-01-13 13:22:05 DEBUG    tasks run_analysis_cleanup: analysis completion status: 'SUCCESS'
2015-01-13 13:22:05 DEBUG    tasks rename_analysis_results: analysis_manager.rename_analysis_results called
2015-01-13 13:22:05 DEBUG    tasks rename: Renaming FileStoreItem with UUID '369b174f-1621-41c6-8d58-98ca602d14d0'
2015-01-13 13:22:05 DEBUG    models rename_datafile: Renaming datafile 7d/3f/display to 1_test tool out.galaxy.datatypes.data.Text
2015-01-13 13:22:05 DEBUG    models _rename_file_on_disk: Renamed /vagrant/media/file_store/7d/3f/display to /vagrant/media/file_store/82/64/1_test tool out.galaxy.datatypes.data_2.Text
2015-01-13 13:22:05 INFO     models rename_datafile: Datafile renamed
```

**Comment by jmchilton** [comment]
It is probably this change (https://bitbucket.org/galaxy/galaxy-central/commits/9d152ed#chg-lib/galaxy/model/__init__.py). I have had to update a couple projects to deal with the fallout already (e.g. https://github.com/jmchilton/blend4j/commit/f9a5a4bd32424f039ee90c9c1735bc5f90a38973 and https://github.com/afgane/bioblend/commit/cd8566cdef6ab6bcec836e00d3f70a9a5ee0ba80).

Hope this helps.

**Comment by hackdna** [comment]
Looks like that would be it. Thanks John!

**Comment by ngehlenborg** [comment]
@hackdna: can you confirm that this does not affect the hbc fork?

**Comment by hackdna** [comment]
It is not an issue with galaxy-central-hbc yet because the last merge was with latest_2014.08.11 in upstream.

**Comment by ngehlenborg** [comment]
@scottx611x: You can test this with the Refinery test workflows.

**Comment by scottx611x** [comment]
@hackdna Could you provide me with some more information on this?
I am able to run a test workflow with real data and see the expected output in the dataset view

**Comment by hackdna** [comment]
What version of Galaxy are you running?

**Comment by scottx611x** [comment]
@hackdna I've switched to the newest galaxy and I'm still able to run the test workflows on real data successfully and see the output in the dataset view

**Comment by scottx611x** [comment]
This has been address as of: c1e50a7ae9da271c2fd96e23912f387640d34c9

**Comment by hackdna** [comment]
The fix is in pull request #908.
